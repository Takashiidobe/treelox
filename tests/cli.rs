use assert_cmd::{assert::OutputAssertExt, cargo::CommandCargoExt};
use insta::{assert_debug_snapshot, glob};
use insta_cmd::Command;

#[test]
fn test_lox() {
    glob!("../test-files", "*.lox", |path| {
        let mut cmd = Command::cargo_bin("treelox").unwrap();

        assert_debug_snapshot!(cmd.arg(path).assert());
    });
}
