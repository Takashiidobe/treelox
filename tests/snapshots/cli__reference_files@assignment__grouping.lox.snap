---
source: tests/cli.rs
expression: test_output
input_file: test-files/assignment/grouping.lox
---
status: 65
stdout: []
stderr:
  - "[line 2] Error at '=': Invalid assignment target."
